### PR TITLE
fix arxiv abstract background

### PR DESCRIPTION
**Pull request by TDHTTTT**

### CODE REVIEW

**Comment by Gusted** [comment]
Hi @TDHTTTT!

Could you provide a example URL?

Regards,
Gusted

**Comment by TDHTTTT** [comment]
https://arxiv.org/abs/2011.01984

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -531,6 +531,15 @@ INVERT
 
 ================================
 
+arxiv.org
+
+CSS
+.abstract {
+     background-color: transparent !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:538]: Why is it being set to `transparent`. It seem good for me without this fix.

Comment (TDHTTTT) [reply src/config/dynamic-theme-fixes.config:538]: Without the fix, the abstract's background is white (https://static.arxiv.org/static/base/0.16.9rc3/css/abs.css)
```
#abs blockquote.abstract {
	font-size: 1.05em;
	margin-block-end: 21.6px;
	margin-block-start: 14.4px;
	margin-bottom: 21.6px;
	margin-inline-end: 40px;
	margin-inline-start: 40px;
	background-color: white;
	border-left: 0px;
	padding: 0px;
}
```

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:538]: It should be darkend by dark reader to the right color.
image

Comment (TDHTTTT) [reply src/config/dynamic-theme-fixes.config:538]: That's weird. What I see is:

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:538]: Any custom settings? Did you try enable/disabling Dark reader.

**Pull request merged**